ci/validate-formula.sh

```diff
@@ -0,0 +1,84 @@
+#! /bin/bash
+
+# this script is intended to be executed via PRs travis CI
+set -e
+
+## this will generate the pillar
+function generate_cluster_pillar {
+  METHOD="$1"
+  cat > test/pillar/cluster.sls <<EOF
+  cluster:
+    name: 'hacluster'
+    $METHOD
+    remove: hana02
+    interface: 'eth1'
+    watchdog:
+      module: softdog
+      device: /dev/watchdog
+    sbd:
+      device: '/dev/vdc'
+    ntp: pool.ntp.org
+    sshkeys:
+      overwrite: true
+      password: linux
+    resource_agents:
+      - SAPHanaSR
+    configure:
+      method: update
+      template:
+        source: /srv/salt/hana/templates/scale_up_resources.j2
+        parameters:
+          sid: prd
+          instance: 00
+          virtual_ip: 192.168.107.50
+          virtual_ip_mask: 24
+          platform: libvirt
+          prefer_takeover: true
+          auto_register: false
+EOF
+}
+
+
+
+echo "==========================================="
+echo " generating pillar and top.sls conf files "
+echo "==========================================="
+
+generate_cluster_pillar "init: 'hana01'"
+cat > top.sls <<EOF
+base:
+  '*':
+    - cluster
+EOF
+
+echo "top.sls and pillar --> DONE"
+echo
+
+
+echo "==========================================="
+echo " Using primary host                        "
+echo "==========================================="
+
+cat >minion <<EOF
+root_dir: $PWD
+id: travis
+EOF
+
```

Comment:
I would add here the grains file to make it easier to understand:
```
cat >grains <<EOF
host: hana01
EOF
```

Reply:
done thx